vendor/test/BUILD.gn

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
I think these should go in `components/brave_rewards` and `components/brave_ads`
`vendor` is deprecated

Reply:
Addressed by 6cf64d97fbed644ab5deebc608523eb0006d4568 and https://github.com/brave/brave-browser/pull/9765/commits/5de3cd89b07ad31a0df6a920d3ee20427e00428b